docs/guides/web-servers/caddy/how-to-install-and-configure-caddy-on-centos-8/index.md

```diff
@@ -36,158 +36,105 @@ aliases: ['/web-servers/caddy/how-to-install-and-configure-caddy-on-centos-8/']
 
-        curl https://getcaddy.com | bash -s personal hook.service
+        sudo dnf install caddy
 
-    Caddy will be installed to your `/usr/local/bin/caddy` directory.
+1. To verify the installation of caddy type:
+       caddy version
+    An output similar to the following appears:
 
-    {{< note >}}
-To learn about Caddy licensing, please read their [blog post on the topic](https://caddyserver.com/v1/blog/announcing-caddy-1_0-caddy-2-caddy-enterprise). In 2017, commercial use of Caddy and their binaries required a license, however, they have recently updated their licensing and commercial licenses are no longer required for their use.
-    {{</ note >}}
+        v2.4.3 h1:Y1FaV2N4WO3rBqxSYA8UZsZTQdN+PwcoOcAiZTM8C0I=
 
-1. Add Caddy to your system's `$PATH`.
+{{< caution >}}
+Caddy has recently changed their [license](https://caddyserver.com/products/licenses). Please read over the license agreement to ensure that you are not violating the license with your project. To use Caddy without a commercial license, you may need to [compile from source](/docs/web-servers/caddy/compile-caddy-from-source).
+{{</ caution >}}
 
-        sudo echo 'export PATH=/usr/local/bin/caddy:$PATH' | sudo tee /etc/profile.d/caddy.sh
+## Allow HTTP and HTTPS Connections
 
-1. Reload your system's profile or log out and SSH back into your Linode.
+Caddy serves websites using HTTP and HTTPS protocols, so you need to allow access to the ports 80, and 443.
 
-        . /etc/profile
+        sudo firewall-cmd --permanent --zone=public --add-service=http
+        sudo firewall-cmd --permanent --zone=public --add-service=https
+        sudo firewall-cmd --reload
+
+## Add Web Content
+
+1.  Set up a home directory, **web root**, for your website:
+
+        sudo mkdir -p /var/www/html/example.com
+
+1. Use SELinux’s `chcon` command to change the file security context for web content:
 
-    {{< note >}}
-You can verify that the Caddy executable is in your system's `$PATH` with the following command:
+        sudo chcon -t httpd_sys_content_t /var/www/example.com -R
+        sudo chcon -t httpd_sys_rw_content_t /var/www/example.com -R
 
-    echo $PATH
+1.  Create a test page:
 
-The output should include the location of your Caddy executable:
+        echo '<!doctype html><head><title>Caddy Test Page</title></head><body><h1>Hello, World!</h1></body></html>' > /var/www/html/example.com/index.html
 
-    /usr/local/bin/caddy:/home/example_user/.local/bin:/home/example_user/bin:/usr/local/bin/caddy:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin
+## Configure the Caddyfile
 
-    {{</ note >}}
+Add your hostname and web root to the Caddy configuration. Use an editor of your choice and replace `:80` with your domain name. Set the root directory of the site to `/var/www/html/example.com` Replace `example.com` with your site's domain name:
 
-1. Install Caddy as a systemd service.
+{{< file "/etc/caddy/Caddyfile" caddy >}}
+example.com {
+root /var/www/html/example.com
```

Comment:
```suggestion
    root /var/www/html/example.com
```

Reply:
Actually:

```
root * /var/www/html/example.com
```

The `*` is necessary because otherwise, `/var/www/html/example.com` would be assumed to be a path matcher by the Caddyfile parser.

See the note under the Syntax section: https://caddyserver.com/docs/caddyfile/directives/root

Reply:
Also, `file_server` should be added. Caddy v1 implicitly had a file server enabled, but that's no longer the case in v2. It needs to be explicitly enabled.